commands/team_test.go

```diff
@@ -108,3 +108,250 @@ func (s *MmctlUnitTestSuite) TestCreateTeamCmd() {
+			Return(mockUser, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			RemoveTeamMember(mockTeam.Id, mockUser.Id).
+			Return(false, &model.Response{Error: nil}).
+			Times(1)
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{mockTeam.Id, mockUser.Id})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Remove users from team by email, no error", func() {
+		printer.Clean()
+		mockTeam := &model.Team{Id: teamArg}
+		mockUser := &model.User{Id: userArg}
+
+		s.client.
+			EXPECT().
+			GetTeam(teamArg, "").
+			Return(mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByEmail(mockUser.Id, "").
+			Return(mockUser, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			RemoveTeamMember(mockTeam.Id, mockUser.Id).
+			Return(false, &model.Response{Error: nil}).
+			Times(1)
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{mockTeam.Id, mockUser.Id})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Remove users from team by username, no error", func() {
+		printer.Clean()
+		mockTeam := &model.Team{Id: teamArg}
+		mockUser := &model.User{Id: userArg}
+
+		s.client.
+			EXPECT().
+			GetTeam(teamArg, "").
+			Return(mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByEmail(mockUser.Id, "").
+			Return(nil, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByUsername(mockUser.Id, "").
+			Return(mockUser, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			RemoveTeamMember(mockTeam.Id, mockUser.Id).
+			Return(false, &model.Response{Error: nil}).
+			Times(1)
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{mockTeam.Id, mockUser.Id})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Remove users from team by user no error", func() {
+		printer.Clean()
+		mockTeam := &model.Team{Id: teamArg}
+		mockUser := &model.User{Id: userArg}
+		s.client.
+			EXPECT().
+			GetTeam(teamArg, "").
+			Return(mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByEmail(mockUser.Id, "").
+			Return(nil, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUserByUsername(mockUser.Id, "").
+			Return(nil, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetUser(mockUser.Id, "").
+			Return(mockUser, nil).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			RemoveTeamMember(mockTeam.Id, mockUser.Id).
+			Return(false, &model.Response{Error: nil}).
+			Times(1)
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{mockTeam.Id, mockUser.Id})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Remove users from team unable to remove error", func() {
```

Comment:
"Remove users from team with an erroneous RemoveTeamMember should return an error"